cli/commands/asset/list.go

```diff
@@ -31,8 +33,22 @@ func ListCommand(cli *cli.SensuCli) *cobra.Command {
 				namespace = types.NamespaceTypeAll
 			}
 
+			fieldSelector, err := cmd.Flags().GetString(flags.FieldSelector)
+			if err != nil {
+				return err
+			}
+
+			labelSelector, err := cmd.Flags().GetString(flags.LabelSelector)
+			if err != nil {
+				return err
+			}
+
+			opts := client.ListOptions{}
+			opts.FieldSelector = fieldSelector
+			opts.LabelSelector = labelSelector
```

Comment:
I feel like we could create a nice wrapper for the 12 lines above since it's basically the same across all commands!

Reply:
You're right, I'll refactor that!
I got a bit carried away with my vim macros ;)